src/move_embargo_args.py

```diff
@@ -234,13 +467,94 @@ def parse_args():
                     register_dataset_types=True,
                     transfer_dimensions=True,
                 )
+        ids_moved = []
+        for dt in dest_registry.queryDatasets(
+            datasetType=datalist_exposure, collections=collections_exposure
+        ):
+            try:
+                ids_moved.append(dt.dataId.mapping["exposure"])
+            except KeyError:
+                continue
+        """
```

Comment:
Since `ids_moved` here is just all the datasets in the destination, it won't work correctly when the destination already has data in it.  I think it's best to just remove this and the equivalent below.

Reply:
I agree, let's remove it for the purposes of testing the deployment